app/index.html

```diff
@@ -9,7 +9,7 @@
 -->
 
 <!DOCTYPE html>
-<html lang="en-US">
+<html lang="en-US" class="no-js">
```

Comment:
To test this, I would have to manually add this class back to the rendered HTML that I'm copying to the archive as part of simulating the pre-rendered output. That means that the actual prerender script would need to do that as well, right?

Reply:
Yes!